pkg/ingester/errors.go

```diff
@@ -22,10 +22,17 @@ import (
 	"github.com/grafana/mimir/pkg/util/validation"
 )
 
+const (
+	unavailable = int(codes.Unavailable)
```

Comment:
nit: I don't think it's worth defining a new constant here instead of just having this inline where it's needed. It causes people to have to jump around the code more to understand what's happening.

Reply:
I have removed the constant.